content/practices/ipm/index.md

```diff
@@ -0,0 +1,137 @@
+tags: ["Delivery"]
+length: "1 Hour"
+participants: "Core delivery team"
+# custom "cover" image example: "boris/boris.png"
+image: "default-cover.png" 
+lastmod: "2021-06-17"
+date: "2021-06-17"
+why: 
+- Regular planning meetings help ensure the product backlog is well-understood by all team members and always reflects the current priorities. By discussing and sizing product backlog items, the team may align on the delivery impact of the work to be done.
+when:
+- Iteration Planning Meetings should generally follow the cadence of product iterations (e.g. weekly) or should be held as often as needed to maintain a well-sized and well-understood product backlog.
+what:
+- A product backlog
+- A prepared set of prioritized user stories to be reviewed and potentially promoted to the product backlog
+
+# If this practice or workshop has a Miro template: remote: true
+remote: false
+miro_template_url: "URL for related Miro template" 
+
+---
+## How to Use this Method
+### Sprint Planning vs. Iteration Planning
+[Scrum “Sprints”](https://en.wikipedia.org/wiki/Scrum_(software_development)#Sprint_planning) and “Iterations” are both timeboxed activities but with some differences that impact how planning is done:
+
+{{< table "table" >}}
+|  | Sprint Planning | Iteration Planning |
+| ---- | ---- | ---- |
+| **Prerequisites** | • A Sprint Goal<br>• A prioritized Product Backlog | • A prioritized Product Backlog<br>• User stories to be reviewed  |
+| **Objective** | • Align on a goal for the Sprint<br>• Negotiate and plan the scope for the Sprint (a.k.a. the Sprint Backlog) | • Maintain a shared understanding of the current scope and priorities (i.e. the Product Backlog) | 
+| **Activities** | • Review user stories<br>• Pull user stories from the Product Backlog to the Sprint Backlog | • Review user stories<br> • Estimate user stories<br> • Promote user stories to the Product Backlog according to priority | 
+| **Outputs** | • An unprioritized Sprint Backlog that the team commits to deliver and demonstrate at the end of the Sprint | • A prioritized Product Backlog<br> • An estimation of the work that may be delivered and demonstrated during the Iteration | 
+| **Participants** | • Product Owner<br> • Engineers | • Core team |
+{{</ table >}}
+
+### Sample Agenda & Prompts
+1. Start by framing the conversation for the meeting. Help to set overall project context as needed by briefly recapping recently accepted and in-progress stories or the current state of the newest capabilities in your acceptance environment, to get everyone aligned on where things currently stand as you enter this next iteration. Review the arc of new user stories by outlining what they will add to the product. Present user interface mockups, if available.
+
+
+1. Read through the first user story to explain the business value, the user value, and the acceptance criteria. Allow participants to ask questions, providing clarifications as needed.
+   
+   {{< callout >}}
```

Comment:
This feels like excessive use of a callout. If you call out too many things you risk them losing their purpose to emphasize something.

I might recommend a simple (ordered or unordered) list for all of these tips.

Reply:
I 100% agree. We have an internal thread about this. Publicly: as part of the [User Research Intro Session](https://tanzu.vmware.com/developer/practices/user-research-introduction-session/), we published a "Dos and Don'ts" companion article; we might be able to do a similar thing for an estimation guide or such.

Reply:
Moved the Estimation tips to their own section under "facilitator tips"